.github/workflows/haskell.yml

```diff
@@ -119,26 +99,7 @@ jobs:
         TMPDIR: ${{ runner.temp }}
         TMP: ${{ runner.temp }}
         KEEP_WORKSPACE: 1
-      run: cabal test all
-
-    - # Delete golden files and regenerate them in the next step to ensure
```

Comment:
I think it may be useful to have that. That's a safeguard against putting non-golden files into golden directory accidentaly.

Reply:
Restored.